### DISCUSSION TITLE
Packaging job was unsuccessful while trying to download QFieldcloud project on qfield app

### DISCUSSION THREAD

**Discussion by duboisAM** [opengisch/QField#3794] [Bugs]
Hello everyone,
I have 3 projects on QFieldCloud. When I try to download the most recent one in QField app (on a windows device or an android device), it gives me an error message saying "download failed because the packaging job was unsuccessful". 

I tried uploading a new copy of the same project and I'm still having the same issue.

I tried uploading a new project completely and that worked! The other 2 projects uploaded before the one having issue are also downloadable without problems.

It's starting to look like it's the project itself. I know my layers (rasters and shapefiles) don't necessarily have the same projection and the project takes nearly 1,5 Go. Is there a space limit for qfieldcloud projects? Does anyone have a clue what could cause the problem?

Kind regards,

**Comment by suricactus** [comment]
Hello, @duboisAM . Having a big projects on QFieldCloud might lead to errors. Currently all projects must be runnable in QGIS with 2GB RAM and all tasks should finish withing 10 minutes. Please let me know your username/projectname combo, so I can give you a more detailed answer.

**Reply by duboisAM** [reply]
@suricactus here are the needed infos :
amdubois (username) / Recolte_qgis_cloud (project name)

**Reply by suricactus** [reply]
@duboisAM In the web UI, click on your project, go to "Jobs" tab and find the latest "Packaging job" there. You will see the last line says:
```
Timeout error! The job failed to finish within 600 seconds!
```

Which basically means you have to make your project slimmer and try again.

**Reply by duboisAM** [reply]
Good to know. But how much slimmer? I didn't read any recommendations as to what is the maximum space one project should use?

**Reply by suricactus** [reply]
This is a good question, but we don't have a strict recommendation. It depends very much on the layers you have in your project and the bandwidth for online layers (e.g. PostgreSQL).  I would recommend to look at the logs of your job and try to remove heaviest parts.
